### PR TITLE
Works and About features

### PR DESCRIPTION
**Pull request by zied2112**
Working on new features for portfolio figma template design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,51 +1,184 @@
...
+              <li>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-angellist"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
```
Comment (franclobo) [inline index.html]: - [ ] Kindly add the rest of the icons (Medium), there are 4 of 5, 👉🏽 we must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

File: index.html [hunk 2]
```diff
@@ -1,51 +1,184 @@
...
+          </div>
+          <div class="project-description">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
```
Comment (franclobo) [inline index.html]: - [ ] Please complete the cards about works there are 3 of 6, review the requirement above. ☝🏽

File: style.css [hunk 3]
```diff
@@ -5,9 +5,12 @@
   box-sizing: border-box;
   margin: 0;
   padding: 0;
-  scroll-behavior: smooth;
-  overflow: hidden;
-  font-family: 'roboto', Arial, Verdana, Geneva, Tahoma, sans-serif;
+  overflow: scroll;
```
Comment (franclobo) [inline style.css]: - [ ] Avoid scrolling the text from the description, for a better user experience. 😄 
- Figma template
image

- Yours

image

File: style.css [hunk 4]
```diff
@@ -85,11 +87,264 @@ ul {
...
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  margin: auto;
+}
```
Comment (franclobo) [inline style.css]: - [ ] Kindly add the background image, review the requirement above. 👆🏽 
- Figma template
image

- Yours:
image

File: style.css [hunk 5]
```diff
@@ -85,11 +87,264 @@ ul {
...
+.about-description > .btn {
+  border-radius: 5px;
+  font-size: 1.3rem;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px;
+  background-color: #32b37e;
+  color: white;
+  margin: 15px 0 25px 0;
+}
```
Comment (franclobo) [inline style.css:287]: - [ ] Please, implement the button interactions (enable, hover, etc.), on the works section and about section, it is for a better user experience. ⭐ 
image

File: index.html [hunk 6]
```diff
@@ -1,51 +1,184 @@
...
+          </div>
+        </div>
+        <div class="project">
+          <div class="project-image">
+            <img src="images/Asset3.png" alt="project1" />
```
Comment (franclobo) [inline index.html]: - [ ] There are no background images in the cards in the Figma template, please the design must be similar to the template. 😳 

- Figma tem plate 
image

- Yours:
image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @zied2112, 

Good job so far!
You did a good job in your portfolio, congratulations!!! ⭐ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️Git/GitHub flow best practices have been followed
✔️There are no linter errors
✔️Desriptive README file
✔️Descriptive commits and PR.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Please, we could avoid using scroll in the components.
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zied2112** [comment]
image

**Review by frank1738 — APPROVED** [review]
Hi @zied2112 , 

Good job implementing the Requested changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

5SW

## Highlights

- Have used the right GitHub flow ✅
- Descriptive Comments ✅
- Professional README.md file ✅
- Linter checks passed ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**